avm/utilities/pipelines/publish/Publish-ModuleFromTagToPBR.ps1

```diff
@@ -1,66 +1,62 @@
+<#
+.SYNOPSIS
+Publish a module based on the provided git tag
+
+.DESCRIPTION
+Publish a module based on the provided git tag
+
+.PARAMETER ModuleReleaseTagName
+Mandatory. The git tag to identify the module with & publish its code state of
+
+.PARAMETER PublicRegistryServer
+Mandatory. The public registry server.
+
+.EXAMPLE
+Publish-ModuleFromTagToPBR -ModuleReleaseTagName 'avm/res/key-vault/vault/0.3.0' -PublicRegistryServer (ConvertTo-SecureString 'myServer' -AsPlainText -Force)
+
+Publish the module 'avm/res/key-vault/vault' of git tag 'avm/res/key-vault/vault/0.3.0' to the public registry server 'myServer'
+#>
 function Publish-ModuleFromTagToPBR {
 
-  [CmdletBinding()]
+  [CmdletBinding(SupportsShouldProcess)]
   param (
-    [Parameter(Mandatory = $true)]
+    [Parameter(Mandatory)]
     [string] $ModuleReleaseTagName,
 
-    [Parameter(Mandatory = $true)]
+    [Parameter(Mandatory)]
     [secureString] $PublicRegistryServer
   )
 
   # Load used functions
-  . (Join-Path $PSScriptRoot 'helper' 'Get-BRMRepositoryName.ps1')
   . (Join-Path $PSScriptRoot 'helper' 'Get-ModuleReadmeLink.ps1')
-  . (Join-Path (Split-Path $PSScriptRoot -Parent) 'sharedScripts' 'tokenReplacement' 'Convert-TokensInFileList.ps1')
-
-  # TODO: Diff in between tag & tag^-1 to find modules to publish?
 
   # 1. Find tag as per function input
-  $repositoryRoot = (Get-Item $PSScriptRoot).Parent.Parent.Parent.Parent.Parent
+  $repositoryRoot = (Get-Item $PSScriptRoot).Parent.Parent.Parent.Parent.Parent.FullName
```

Comment:
As per what discussed some time ago, I'd suggest a repositoryroot input parameter with default value, and passing the repo root from the pipeline, leveraging the corrwsponding gh env var

Reply:
Added to both publish scripts. While I fundamentally think it's a good idea - I'd like to discuss this if we want to do this on scale. On the upside, it allows you to provide whatever path you want in case you want to load dependent functions in isoluation from a custom location.
The downside I'd like to bring up is that we're introducing more and more dependencies on the exact folder structure (outside of e.g. the publishing). I.e., we're using now absolute paths instead of relative ones. If we were to rename e.g. the pipelines folder or something, we'd need to now update a lot more places than before. Not saying we shouldn't do this necessarily, but I'd at least like to question if we're doing us a favor.

Reply:
But they're not. We're just passing the repository root. If we'd want to make them less dependent, we'd need to pass each function's path.

Reply:
That is true. It would however result in less effort through search&replace vs wondering how many levels to go up and down in the hierarchy starting from the current script.

Reply:
That is true.